mod error;

pub use self::error::{Error, Result};

fn main() {
    println!("Hello, world!");
}
